src/powerquery-language-services/inspection/activeNode/activeNodeUtils.ts

```diff
@@ -16,6 +16,8 @@ import type { Position } from "vscode-languageserver-types";
 import { ActiveNode, ActiveNodeKind, ActiveNodeLeafKind, OutOfBoundPosition, TMaybeActiveNode } from "./activeNode";
 import { PositionUtils } from "../..";
 
+type MaybeAstIdentifier = Ast.Identifier | Ast.GeneralizedIdentifier | undefined;
```

Comment:
Stick to either `MaybeAstIdentifier` or `Ast.Identifier | Ast.GeneralizedIdentifier | undefined`. I'd lean the more verbose typing as it's one less indirection for new readers of the code, but I don't have a strong opinion on it. If you do keep the typing then make sure it's after any exports.

Reply:
Yep, we should find its better place within the parser next time 😀

Reply:
Then for now let's remove this typing and replace the typings below with their more verbose counterparts.